Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/ui/list/adapter/items/VaccinationListCertificateCardItem.kt

```diff
@@ -0,0 +1,44 @@
+package de.rki.coronawarnapp.vaccination.ui.list.adapter.items
+
+import android.view.ViewGroup
+import androidx.core.view.isVisible
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.databinding.VaccinationListCertificateCardBinding
+import de.rki.coronawarnapp.presencetracing.checkins.qrcode.QrCodeGenerator
+import de.rki.coronawarnapp.vaccination.ui.list.adapter.VaccinationListAdapter
+import de.rki.coronawarnapp.vaccination.ui.list.adapter.VaccinationListItem
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+data class VaccinationListCertificateCardItem(
+    val qrCodeData: String,
```

Comment:
Subjective, I prefer the data class to be inlined as it seems easier to read/relate (and Android Studio doesn't fail at refactoring as much as when there are multiple top level classes in a file.

Reply:
Makes sense, I refactored the items.